Stateczki/Oceans/OceanDisplayer.cs

```diff
@@ -6,5 +6,99 @@ namespace Stateczki
 {
     static class OceanDisplayer
     {
+        private static Dictionary<string, string> symbols = new Dictionary<string, string>
+        {
+            {"ship", "S" },
+            {"shoot", "X" },
+            {"empty", "O" }
+        };
+
+        public static void PrintOceanForPlay(Square[,] ocean)
```

Comment:
As above, after meging PR #2 we can reference Ocean Type from here. (And the next method)